js/main.js

```diff
@@ -21,4 +20,90 @@ function submitEntry(event) {
   data.entries.unshift(entryObj);
   image.setAttribute('src', 'images/placeholder-image-square.jpg');
   formEntry.reset();
+  ulEl.prepend(renderList(entryObj));
+  $getForm.className = 'hidden';
+  $getEntries.className = 'get-entries';
+  var $noDiv = document.querySelector('.no-entries');
+  $noDiv.className = 'no-entries hidden';
+  data.view = 'entries';
 }
+
+function renderList(entry) {
+  var liEl = document.createElement('li');
+  liEl.setAttribute('class', 'row margin-b40 align-center');
+
+  var divImg = document.createElement('div');
+  divImg.setAttribute('class', 'column-full column-half');
+  liEl.appendChild(divImg);
+
+  var viewImg = document.createElement('img');
+  viewImg.setAttribute('src', entry.photoUrl);
+  divImg.appendChild(viewImg);
+
+  var divText = document.createElement('div');
+  divText.setAttribute('class', 'column-full column-half');
+  liEl.appendChild(divText);
+
+  var h3Title = document.createElement('h3');
+  h3Title.textContent = entry.title;
+  divText.appendChild(h3Title);
+
+  var paraNotes = document.createElement('p');
+  paraNotes.textContent = entry.notes;
+  divText.appendChild(paraNotes);
+
+  return liEl;
+}
+
+var ulEl = document.querySelector('.list-none');
+window.addEventListener('DOMContentLoaded', loadView);
+
+function loadView(event) {
+  if (data.view === 'entry-form') {
+    $getForm.className = 'get-entries';
+    $getEntries.className = 'hidden';
+  } else if (data.view === 'entries') {
+    $getForm.className = 'hidden';
+    $getEntries.className = 'get-entries';
+    hideFormShowEntries();
+  }
+
+  for (var i = data.entries.length - 1; i >= 0; i--) {
+    ulEl.prepend(renderList(data.entries[i]));
+  }
+}
+
+var viewEntries = document.querySelector('.entry-view');
+var $getEntries = document.querySelector('.get-entries');
+var $getForm = document.querySelector('.get-form');
+viewEntries.addEventListener('click', hideFormShowEntries);
+
+function hideFormShowEntries(event) {
+  $getForm.className = 'hidden';
+  $getEntries.className = 'get-entries';
+  data.view = 'entries';
+}
+
+var $newEntry = document.querySelector('.new-entry');
+$newEntry.addEventListener('click', showFormHideEntries);
+
+function showFormHideEntries(event) {
+  $getForm.className = 'get-entries';
+  $getEntries.className = 'hidden';
+  data.view = 'entry-form';
+
+}
```

Comment:
This is a very manual way of handling view swapping, I suggest you try to implement a system more similar to what you did in the `javascript-view-swapping` exercise that used a loop over all view containers.